Travis:
```
[ERROR] /home/travis/build/Sergix/JTerm/src/main/java/jterm/JTerm.java:[164,22] cannot find symbol
  symbol:   class InvocationTargetException
  location: class jterm.JTerm
```

--- a/src/main/java/jterm/JTerm.java
+++ b/src/main/java/jterm/JTerm.java
@@ -17,6 +17,7 @@
 // package = folder :P
 package jterm;
 
+import java.lang.reflect.InvocationTargetException;
 import jterm.command.Command;
 import jterm.command.CommandException;
 import jterm.command.CommandExecutor;

